src/components/filters/FilterSection/index.jsx

```diff
@@ -73,6 +76,27 @@ class FilterSection extends React.Component {
     );
   }
 
+  getAndOrToggle() {
+    const isHidden = !this.state.showingAndOrToggle || !this.state.isExpanded;
+    const isAndMode = this.state.combineMode === 'AND';
+    return (
+      <div className={`g3-filter-section__and-or-toggle ${isHidden && 'g3-filter-section__hidden'}`}>
+        <span className='g3-filter-section__combine_label'>Combine with:</span>
+        <button type='button' onClick={() => this.handleSetCombineModeOption('AND')} className={`${isAndMode && 'g3-filter-section__and_or_active'}`}>AND</button>
+        <button type='button' onClick={() => this.handleSetCombineModeOption('OR')} className={`${!isAndMode && 'g3-filter-section__and_or_active'}`}>OR</button>
```

Comment:
for the visual presentation these 2 button, i would suggest we stick to the same color patterns as our filter components: which means the selected/activated ones should have the prime color, rather than just light gray